feature-libs/product-configurator/rulebased/components/form/configurator-form.component.ts

```diff
@@ -45,120 +50,111 @@ export class ConfiguratorFormComponent implements OnInit {
-    this.configuratorCommonsService.updateConfiguration(
-      event.ownerKey,
-      event.changedAttribute,
-      event.updateType
-    );
+    this.routerData$.pipe(take(1)).subscribe((routingData) => {
+      //In case of resolving issues (if no conflict solver dialog is present!), check if the configuration contains conflicts,
+      //if not, check if the configuration contains missing mandatory fields and show the group
+      if (routingData.resolveIssues) {
+        this.configuratorCommonsService
+          .hasConflicts(routingData.owner)
+          .pipe(take(1))
+          .subscribe((hasConflicts) => {
+            if (hasConflicts && !routingData.skipConflicts) {
+              this.configuratorGroupsService.navigateToConflictSolver(
+                routingData.owner
+              );
+
+              //Only check for Incomplete group when there are no conflicts or conflicts should be skipped
+            } else {
+              this.configuratorGroupsService.navigateToFirstIncompleteGroup(
+                routingData.owner
+              );
+            }
+          });
+      }
+
+      if (routingData.expMode) {
+        this.configExpertModeService?.setExpModeRequested(routingData.expMode);
+      }
+      this.conflictSolverSubscription = this.configuratorCommonsService
+        .getConfiguration(routingData.owner)
+        .pipe(
+          distinctUntilChanged(
+            (a, b) =>
+              a.interactionState.showConflictSolverDialog ===
+              b.interactionState.showConflictSolverDialog
+          )
+        )
+        .subscribe((config) => {
+          if (config.interactionState.showConflictSolverDialog) {
+            if (
+              config.interactionState.currentGroup?.startsWith(
+                Configurator.ConflictIdPrefix
+              )
+            ) {
+              this.configuratorGroupsService.navigateToFirstAttributeGroup(
+                config
+              );
+            }
+          }
+        });
+    });
   }
 
-  isConflictGroupType(groupType: Configurator.GroupType): boolean {
-    return this.configuratorGroupsService.isConflictGroupType(groupType);
+  ngOnDestroy(): void {
+    this.conflictSolverSubscription?.unsubscribe();
   }
 
   /**
-   * Display group description box only for conflict groups with a given group name (i.e. a conflict description)
-   * @param group
-   * @returns true if conflict description box should be displayed
+   * Verifies whether the navigation to a conflict group is enabled.
+   *
+   * @returns {Observable<boolean>} Returns 'true' if the navigation to a conflict group is enabled, otherwise 'false'.
    */
-  displayConflictDescription(group: Configurator.Group): boolean {
-    return (
-      group.groupType !== undefined &&
-      this.configuratorGroupsService.isConflictGroupType(group.groupType) &&
-      group.name !== ''
+  isNavigationToGroupEnabled(): Observable<boolean> {
```

Comment:
Wouldn't be "isImmediateConflictResolutionEnabled" a better name? This method/information is not only used for checking whether group navigation is enabled but more generally whether the immediate conflict resolution is enabled.